src/main/resources/application.properties

```diff
@@ -71,7 +71,7 @@ spring.jpa.show-sql=false
 spring.jpa.hibernate.ddl-auto=none
 
 # Flyway (db migration)
-flyway.locations=sql
+flyway.locations=src/main/resources/sql
```

Comment:
This will not work, right? `src/main/resources/sql` does not look like a classpath location